app/src/main/kotlin/com/mapbox/vision/teaser/replayer/ReplayModeFragment.kt

```diff
@@ -0,0 +1,190 @@
+package com.mapbox.vision.teaser.replayer
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.*
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.mapbox.vision.teaser.OnBackPressedListener
+import com.mapbox.vision.teaser.R
+import com.mapbox.vision.teaser.utils.FileUtils
+import kotlinx.android.synthetic.main.fragment_replay_mode.*
+import java.io.File
+import java.lang.RuntimeException
+
+class ReplayModeFragment : Fragment(), OnBackPressedListener {
+
+    private var sessionsAdapter: SessionsAdapter? = null
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.fragment_replay_mode, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initBackButton()
+        initRecyclerViewSessions()
+        initSelectAllButton()
+        initEditButton()
+        initSwipeRefreshLayout()
+        initDeleteSessionsButton()
+        initDoneEditButton()
+    }
+
+    private fun initBackButton() {
+        back_button.setOnClickListener {
+            requireActivity().onBackPressed()
+        }
+    }
+
+    private fun initRecyclerViewSessions() {
+        recycler_sessions.layoutManager = LinearLayoutManager(requireContext())
+        val sessionsPath = arguments?.getString(ARG_PARAM_SESSIONS_PATH)
+        if (sessionsPath != null) {
+            sessionsAdapter = SessionsAdapter(
+                    requireContext(),
```

Comment:
Pls add names.
Also, continuation indent seems 8 spaces instead of 4

Reply:
Will change, I've used hotkeys for default AS alignment

Reply:
fixed